package.json

```diff
@@ -12,29 +12,29 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^15.1.3",
-    "@angular/common": "^15.1.3",
-    "@angular/compiler": "^15.1.3",
-    "@angular/core": "^15.1.3",
-    "@angular/forms": "^15.1.3",
-    "@angular/platform-browser": "^15.1.3",
-    "@angular/platform-browser-dynamic": "^15.1.3",
-    "@angular/router": "^15.1.3",
+    "@angular/animations": "^16.0.0",
+    "@angular/common": "^16.0.0",
+    "@angular/compiler": "^16.0.0",
+    "@angular/core": "^16.0.0",
+    "@angular/forms": "^16.0.0",
+    "@angular/platform-browser": "^16.0.0",
+    "@angular/platform-browser-dynamic": "^16.0.0",
+    "@angular/router": "^16.0.0",
     "core-js": "^3.19.3",
     "rxjs": "^6.5.5",
     "tslib": "^2.2.0",
-    "zone.js": "~0.11.4"
+    "zone.js": "^0.13.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^15.1.4",
-    "@angular/cli": "^15.1.4",
-    "@angular/compiler-cli": "^15.1.3",
-    "@angular/language-service": "^15.1.3",
-    "@angular-eslint/builder": "^15.2.0",
-    "@angular-eslint/eslint-plugin": "^15.2.0",
-    "@angular-eslint/eslint-plugin-template": "^15.2.0",
-    "@angular-eslint/schematics": "^15.2.0",
-    "@angular-eslint/template-parser": "^15.2.0",
+    "@angular-devkit/build-angular": "^16.0.0",
+    "@angular-eslint/builder": "^16.0.1",
+    "@angular-eslint/eslint-plugin": "^16.0.1",
+    "@angular-eslint/eslint-plugin-template": "^16.0.1",
+    "@angular-eslint/schematics": "^16.0.1",
+    "@angular-eslint/template-parser": "^16.0.1",
+    "@angular/cli": "^16.0.0",
+    "@angular/compiler-cli": "^16.0.0",
+    "@angular/language-service": "^16.0.0",
```

Comment:
Angular Language Service is not needed to be present in the `package.json` file anymore.